e2e_tests/test_admin.py

```diff
@@ -0,0 +1,169 @@
+    add_task(page, "Personal Einstellen", sector2)
+
+    msglist = page.locator(".messagelist")
+    expect(msglist).to_contain_text(
+        "„Personal Einstellen“ wurde erfolgreich hinzugefügt."
+    )
+
+    page.close()
+
+
+def test_should_not_allow_add_when_same_title_in_same_sector(page: Page, admin_login):
+    uid = str(uuid.uuid4())
+
+    sector1 = "Admin Test " + uid + " 1"
+    add_task(page, sector1)
+    add_task(page, "Personal Einstellen", sector1)
+    add_task(page, "personal einstellen", sector1)
+
+    errlist = page.locator(".errorlist")
+    expect(errlist.get_by_role("listitem")).to_contain_text(
+        "Das kollidiert mit einem anderen Eintrag."
+    )
+
+    page.close()
+
+
+def drag_task_from_to(page, source, target):
+    target_locator = page.locator(
+        f"//*[@class='field-title' and contains(., '{target}')]"
+    )
+
+    drag_handler = page.locator(
+        f"//th[@class='field-title' and contains(., '{source}')]/preceding-sibling::td[@class='drag-handler']/span"
+    )
+
+    drag_handler.drag_to(
+        target_locator,
+        target_position={"x": 10, "y": 50},
+        force=True,
+        timeout=1000,
+    )
+
+
+def test_should_move_and_adjust_slugs_when_dragged(page: Page, admin_login):
+    uid = str(uuid.uuid4())
+
+    sector1 = "Admin Test 1 " + uid
+    sector2 = "Admin Test 2 " + uid
+    task = "To be dragged " + uid
+    sub_task = "Has to be adjusted " + uid
+    sub_sub_task = "Has to be adjusted " + uid
+
+    slug_s1 = slugify(sector1)
+    slug_s2 = slugify(sector2)
+    sub_sub_slug = (
+        "/" + slugify(task) + "/" + slugify(sub_task) + "/" + slugify(sub_sub_task)
+    )
+
+    add_task(page, sector1)
+    add_task(page, task, sector1)
+    add_task(page, sub_task, task)
+    add_task(page, sub_sub_task, sub_task)
+
+    add_task(page, sector2)
+
+    # Make sure, all are visible and not hidden on page 2.
+    page.goto("/admin/cpmonitor/task/?all=&city__id__exact=1")
+
+    # Without exact=True this would also catch the sub-tasks:
+    expect(page.get_by_text(slug_s1 + "/" + slugify(task), exact=True)).to_be_visible()
+    # ...on the other hand, this should catch any sub-tasks:
+    expect(page.get_by_text(slug_s2 + "/" + slugify(task))).not_to_be_visible()
+    expect(page.get_by_text(slug_s1 + sub_sub_slug)).to_be_visible()
+    expect(page.get_by_text(slug_s2 + sub_sub_slug)).not_to_be_visible()
+
+    drag_task_from_to(page, task, sector2)
+
+    expect(page.locator(".messagelist")).to_contain_text("positioniert unterhalb von")
+
+    expect(page.get_by_text(slug_s1 + "/" + slugify(task))).not_to_be_visible()
+    expect(page.get_by_text(slug_s2 + "/" + slugify(task), exact=True)).to_be_visible()
+    expect(page.get_by_text(slug_s1 + sub_sub_slug)).not_to_be_visible()
+    expect(page.get_by_text(slug_s2 + sub_sub_slug)).to_be_visible()
+
+    page.close()
+
+
+def test_should_not_allow_move_when_same_title_in_same_sector(page: Page, admin_login):
```

Comment:
You're additionally testing that case is ignored for comparison, so I'd add that to the title:
```suggestion
def test_should_not_allow_move_when_same_case_ignored_title_in_same_sector(page: Page, admin_login):
```